src/Controls/src/Core/Shadow.cs

```diff
@@ -19,6 +19,13 @@ public class Shadow : Element, IShadow
 
 		Paint IShadow.Paint => Brush;
 
+		internal MergedStyle _mergedStyle;
+
+		public Shadow()
```

Comment:
this doesn't look like the right fix. The problem probably comes from the Shadow isn't properly parented.
(parenting code was probably removed to 'fix' a memory leak, but we have a PR avoiding that leak)

Reply:
@StephaneDelcroix I tried a solution with setting the parent visual element as a parent here: https://github.com/dotnet/maui/blob/5d6753110f98f5997b46e06c920eb341f8381386/src/Controls/src/Core/VisualElement/VisualElement.cs#L309-L325

but it didn't work :/

it's why I did the same thing as, for example, Span does
https://github.com/dotnet/maui/blob/5d6753110f98f5997b46e06c920eb341f8381386/src/Controls/src/Core/Span.cs#L12-L18